src/command/commands.rs

```diff
@@ -182,11 +182,17 @@ impl Command for CreatePlaceholders<'_> {
     fn build(&self) -> CF_OPERATION_PARAMETERS_0 {
         CF_OPERATION_PARAMETERS_0 {
             TransferPlaceholders: CF_OPERATION_PARAMETERS_0_7 {
-                Flags: CloudFilters::CF_OPERATION_TRANSFER_PLACEHOLDERS_FLAG_NONE,
+                Flags: CloudFilters::CF_OPERATION_TRANSFER_PLACEHOLDERS_FLAG_DISABLE_ON_DEMAND_POPULATION,
```

Comment:
I don't think this flag is safe to assume for every call? Should we maybe add a bool variable to the struct?

Reply:
I think it makes sense to prevent the callback from being called multiple times for the same dir, less complication, but this flag also sets a directory as being marked "full." Does that have any side-effects?

If not, I'd agree for this flag to be the default, otherwise it should only be set when PlaceholderCount=0.

Reply:
It's a temporary solution which assume `pass_with_placeholders` will be called once. Maybe it can be refactor to futures::Stream.

You can check: <https://learn.microsoft.com/en-us/windows/win32/api/cfapi/ne-cfapi-cf_operation_transfer_placeholders_flags>

Reply:
I see, that makes sense